style.css

```diff
@@ -504,3 +871,226 @@
   line-height: 2.75rem;
   color: #fff;
 }
+
+@media screen and (min-width: 768px) {
+  .skills-container {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: center;
+    padding-top: 4rem;
+    padding-right: 3rem;
+    padding-left: 2rem;
+  }
+
+  .skills1 {
+    padding-top: 0;
+    padding-left: 1rem;
+    padding-right: 1rem;
+  }
+
+  .skills2 {
+    padding-left: 2rem;
+    padding-right: 2rem;
+  }
+
+  .list-items {
+    text-align: start;
+  }
+
+  .list-items li a {
+    font-weight: 100;
+  }
+}
+
+/* form */
+
+.bottom {
+  display: flex;
+  flex-direction: column;
+  padding-top: 7.5rem;
+  padding-bottom: 7.5rem;
+  padding-left: 1.475rem;
+  padding-right: 1.5625rem;
+  gap: 3.62rem;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement as your design not matching properly with the Figma design, it seems like your form is not aligned properly and not matching with the Figma, which makes your design `inconsistent` and looks unprofessional. Please align the form inputs properly and also use the exact colour given in *Figma* to make your design looks similar to the original have a look at the below-attached design comparison image and table carefully.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/66476329/e6a284fe-3989-4302-bb73-8c15bee1ae58)

</td>
<td>

![Screenshot from 2023-06-17 13-31-51](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/66476329/814ed1f5-53a5-4fae-baa0-a320a347caac)

</td>
</tr>
</table>